test/refactor_nrepl/ns/suggest_aliases_test.clj

```diff
@@ -21,6 +21,22 @@
     'unit.foo   true
     'foo.unit   true))
 
+(deftest readable-as-symbol?
+  (are [input expected] (is (= expected
```

Comment:
no `is`

Reply:
Oh yes, that was a lazy hack to see the failing line `expected: (is (= false (sut/readable-as-symbol? "ok.one.more!")))` vs `expected: false`. I put in a proper fix: a plain `(readable-as-symbol? expected)` does the trick vs `(is (= true (...)))`.

Reply:
Yep, I understand!

That's why there's a `true` at tail position in this ns's `are`s (kondo and Eastwood won't complain - they understand the trick)